probes/http/request.go

```diff
@@ -101,5 +105,9 @@ func (p *Probe) httpRequestForTarget(target endpoint.Endpoint, resolveF resolveF
 		req.Header.Set("Authorization", "Bearer "+p.bearerToken)
 	}
 
+	if len(body) >= largeBodyThreshold {
```

Comment:
Please refer the to comment above. I don't think we need to keep request body bytes per request.

Reply:
Done as above.